packages/lib-classifier/src/store/Subject/Subject.js

```diff
@@ -80,8 +80,12 @@ const Subject = types
       return tryReference(() => getRoot(self).workflows?.active)
     },
 
-    get priority () {
-      return self.metadata['#priority'] ?? self.metadata.priority
+    get priority() {
+      const priority = self.metadata['#priority'] ?? self.metadata.priority
+      if (priority !== undefined) {
+        return parseFloat(priority)
+      }
+      return undefined
```

Comment:
There should be a spec for this.

Reply:
Yeah. I'm thinking about moving the bug fixes into their own PR. `workflow.prioritized` isn't supported on master either, so I fixed that here, which in turn meant updating all the tests that mock prioritised subjects with workflows that aren't prioritised.

That would mean some extra review work, on top of this PR, ad the tests here will fail until those fixes are in.

Reply:
OK, #2414 fixes `workflow.prioritized` and `subject.priority`, which are both slightly broken at the moment.

Reply:
And that's rebased into this branch now.